PubNub/Core/PubNub+Objects.m

```diff
@@ -514,284 +438,487 @@ - (void)sendFetchChannelMembersRequestUsingBuilderParameters:(NSDictionary *)par
 
 #pragma mark - UUID metadata object
 
-- (void)setUUIDMetadataWithRequest:(PNSetUUIDMetadataRequest *)request
-                        completion:(PNSetUUIDMetadataCompletionBlock)block {
-
-    request.identifier = request.identifier.length ? request.identifier : self.configuration.userID;
-    __weak __typeof(self) weakSelf = self;
-    
-    [self performRequest:request withCompletion:^(PNSetUUIDMetadataStatus *status) {
-        if (block && status.isError) {
-            status.retryBlock = ^{
-                [weakSelf setUUIDMetadataWithRequest:request completion:block];
+- (void)setUUIDMetadataWithRequest:(PNSetUUIDMetadataRequest *)userRequest
+                        completion:(PNSetUUIDMetadataCompletionBlock)handleBlock {
+    userRequest.identifier = userRequest.identifier.length ? userRequest.identifier : self.configuration.userID;
+    PNOperationDataParser *responseParser = [self parserWithStatus:[PNSetUUIDMetadataStatus class]];
+    PNSetUUIDMetadataCompletionBlock block = [handleBlock copy];
+    PNParsedRequestCompletionBlock handler; 
+
+#ifndef PUBNUB_DISABLE_LOGGER
```

Comment:
Looks like it's not used

Reply:
I can drop it. Just added as a placeholder (it still will be stripped away during compilation).